in_toto/model.go

```diff
@@ -1070,11 +1070,11 @@ It wraps the generic SSL envelope signer and enforces the correct payload
 type both during signature generation and validation.
 */
 type SSLSigner struct {
```

Comment:
We should rename this and modify its docstring.

Reply:
Renaming the SSLSigner struct would break compatibility to older versions. Do we really want to do this?

Reply:
https://github.com/golang/go/wiki/Deprecated

Reply:
I think using deprecated is a good idea here.. IMHO the whole struct should be replaced via an interface anyway (for easier test mockups).

Reply:
A bunch of the patches have been merged now! I think we can proceed with this PR, with `DSSESigner` added in.